odoo/addons/base/models/ir_ui_view.py

```diff
@@ -460,6 +460,8 @@ def _compute_defaults(self, values):
     def _compute_warning_info(self):
         for view in self:
             view.warning_info = ''
+            if not view.id:
+                continue
```

Comment:
IMO, it is nice to be able to see the warning during the creation of the ir.view (not only after the save).
Why not making `_get_inheriting_views` robust to NewId ? 
```python
if not self.ids:
     return self.browse()
```

Reply:
Hello @ryv-odoo :wave: 
Requested changes are done.
Please have a look :slightly_smiling_face: 
Thanks!